source/modulo_components/include/modulo_components/Component.hpp

```diff
@@ -58,11 +58,12 @@ class Component : public ComponentInterface<rclcpp::Node> {
    * @param data Data to transmit on the output signal
    * @param default_topic If set, the default value for the topic name to use
    * @param fixed_topic If true, the topic name of the output signal is fixed
+   * @param publish_manually If true, the output publishing has to be triggered manually
```

Comment:
nitpick (medium): I would avoid the use of the word "manually" since things still occur programmatically. To me it's a bit clearer if the logic is reversed

```suggestion
   * @param publish_on_step If true, the output is published periodically on step.
   * If false, the output is only published on publish_output().
```